Gems/ScriptCanvas/Code/Include/ScriptCanvas/AutoGen/ScriptCanvasFunction_Header.jinja

```diff
@@ -0,0 +1,66 @@
+{#
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+#}
+
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//
+// This code was produced with AzAutoGen, any modifications made will not be preserved.
+// If you need to modify this code see:
+//
+// Gems\ScriptCanvas\Code\Include\ScriptCanvas\AutoGen\ScriptCanvasFunction_Header.jinja
```

Comment:
Gems should not assume they will be in the engine root.
So a better reference for the path is to use a placeholder such as <scriptcanvas-gem-root>

Also forward slashes should be used since that works everywhere including Windows.
```suggestion
// <scriptcanvas-gem-root>/Code/Include/ScriptCanvas/AutoGen/ScriptCanvasFunction_Header.jinja
```